### PR TITLE
Update multi-proofs.md

### PR DESCRIPTION
**Pull request by 0xjunger**

### CODE REVIEW

**Review by d1onys1us — APPROVED** [review]

**Comment by d1onys1us** [comment]
@CeciliaZ030 can you please proof-read for correctness? we can merge it after that.

File: src/content/docs/core-concepts/multi-proofs.md [hunk 1]
```diff
@@ -4,3 +4,9 @@ description: Core concept page for "Multi-proofs".
 ---
 
 A great resource to learn about Taiko's approach to security with multi-proofs is the Twitter thread [here](https://x.com/taikoxyz/status/1745546868028068273).
+
+Cryptographic implementations are complex and not yet mature. To minimize potential bugs and vulnerabilities, diversity in proof systems is needed. Taiko is one of the advocates who strongly defends the multi-proofs approach in rollups. Taiko's approach aims to increase security and diversity by using multiple proof systems and clients, thus reducing the risk associated with bugs or vulnerabilities in any single system. The approach also includes the integration of different types of proofs.
+
+Taiko proposes a robust multi-proof pipeline by translating assembly-level instructions coming from different execution clients into arithmetizations for algebraic or polynomial proof systems. Different backends to encode these arithmetizations, such as SuperNova, Halo2, and eSTARK can also be used, without being limited by using a single protocol.
```
Comment (CeciliaZ030) [inline src/content/docs/core-concepts/multi-proofs.md:10]:
```suggestion
Taiko proposes a robust multi-proof pipeline with [powdr labs](https://www.powdr.org/), which translats assembly-level instructions coming from different execution clients into arithmetizations for algebraic or polynomial proof systems. Different backends to encode these arithmetizations, such as SuperNova, Halo2, and eSTARK can also be used, without being limited by using a single protocol.
```
Gotta mention powdr here cuz the modular ZkVM is their project, we just use it in our proving stack 😅

File: src/content/docs/core-concepts/multi-proofs.md [hunk 2]
```diff
@@ -4,3 +4,9 @@ description: Core concept page for "Multi-proofs".
...
+
+In addition to ZK proofs, Taiko utilizes SGX (a Trusted Execution Environment developed by Intel) to generate a different type of proof. SGX runs the same code that would be executed on a zkVM, which functions somewhat like a light execution client. Therefore, in contrast to a standard client such as Geth, which requires access to the entire chain state for transaction execution, the proving systems employed by Taiko only need the data essential for verifying blocks. This approach ensures that only necessary data is used and its correctness is verified in conjunction with transaction execution. The necessary data is signed within SGX using a standard ECDSA signature, employing a private key exclusive to SGX. The signature is then verified within the smart contract.
```
Comment (CeciliaZ030) [inline src/content/docs/core-concepts/multi-proofs.md:12]:
```suggestion
In addition to ZK proofs, Taiko utilizes SGX (a Trusted Execution Environment developed by Intel) to generate a different type of proof. SGX runs the same code that would be executed on a zkVM, which functions somewhat like a light execution client. Therefore, all proof systems verify the same underlying light client's execution, potentially allowing for the reuse of necessary data. The necessary data is signed within SGX using a standard ECDSA signature, employing a private key exclusive to SGX. The signature is then verified within the smart contract.
```
The fact that "Taiko only need the data essential for verifying blocks" is not unique to Taiko, all ZK proof works like this. The purpose of our architecture is to proof the same compiled artifacts, and potentially the same execution trace. The trivia is different proofs requires different data format, but the *content* of ZK input should be the same, so we serve them exactly the same.

**Review by CeciliaZ030 — CHANGES_REQUESTED** [review]
@0xjunger Thanks a lot for your input! would be great if you change them a little :)

**Comment by 0xjunger** [comment]
All is done for now. We can improve over time. Thank you so much @CeciliaZ030 @d1onys1us .

**Review by CeciliaZ030 — APPROVED** [review]

**Pull request merged**